### ISSUE TITLE
How to check the Sofia version in FreeSWITCH

### ISSUE THREAD

**Issue by ChiMuYuan** [freeswitch/sofia-sip#239]
Thank you

**Comment by andywolk** [comment]
It's a library that's installed when you install FreeSWITCH. You need to check your system.
on Debian:
```
apt-cache policy libsofia-sip-ua0
```

**Comment by ChiMuYuan** [comment]
Thank you reply.

I'm sorry if I confused you.
I got an error when installing FreeSWITCH using apt, but I succeeded by building it from source.
So, when I check the version using `apt-cache policy libsofia-sip-ua0` it shows `Installed: (none)` 

![image](https://github.com/freeswitch/sofia-sip/assets/26971626/fd36434c-8d97-40bc-82b8-d17492518428)

I saw pull request #233 fixing numerous errors related to WSS, which might be what I need. So, I fetched the code for v1.13.17 and reinstalled it. However, I'm not sure if the installation was successful or if FreeSWITCH is using the latest sofia.
```
git clone https://github.com/freeswitch/sofia-sip.git
git checkout v1.13.17
./bootstrap
./configure
make
make install
```

I'm a beginner, so I apologize for any mistakes.
